From 35ec0dbd47c24ecb43c1fb8b16d256cb3f805ccf Mon Sep 17 00:00:00 2001
From: Pavol Juhas <pavol.juhas@gmail.com>
Date: Tue, 30 Jan 2018 15:36:48 -0500
Subject: BLD: in scons convert python output to strings

SCons runs target python to determine its version and build
flags.  Process such output as strings and not as `bytes`.
---
 SConstruct | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/SConstruct b/SConstruct
index 9b14d44..e687a0c 100644
--- a/SConstruct
+++ b/SConstruct
@@ -34,7 +34,8 @@ def getsyspaths(*names):
 
 def pyoutput(cmd):
     proc = subprocess.Popen([env['python'], '-c', cmd],
-                            stdout=subprocess.PIPE)
+                            stdout=subprocess.PIPE,
+                            universal_newlines=True)
     out = proc.communicate()[0]
     return out.rstrip()
 
-- 
2.16.0

